inst/chunks/fun_mixcure_lognormal.stan

```diff
@@ -0,0 +1,30 @@
+/* mixcure lognormal (AFT) log-PDF of a single response
+ * identity parameterization of the incidence part
+ */
+real mixcure_lognormal_lpdf(real y, real mu, real sigma, real inc) {
```

Comment:
can you add some empty lines between function definitions in both stan files? And add a little more doc at the top?